tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -194,12 +231,14 @@ def run_metadata_impl(self, run, tag):
 
   @wrappers.Request.application
   def info_route(self, request):
-    info = self.info_impl()
+    experiment = plugin_util.experiment_id(request.environ)
```

Comment:
CI (`flake8`) complaining that `plugin_util` isn’t imported?

Reply:
Ah there were a bunch of other build failures too--I was inadvertently not testing everything before.  Fixed.